WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -232,7 +240,7 @@ private extension BlazeCampaignCreationForm {
         .padding(.vertical, Layout.contentPadding)
     }
 
-    func detailView(title: String, content: String, action: @escaping () -> Void) -> some View {
+    func detailView(title: String, content: String, singleLineSubtitle: Bool = false, action: @escaping () -> Void) -> some View {
```

Comment:
nit: Consider renaming this into `isContentSingleLine` to match the other `content` parameter. 
```suggestion
    func detailView(title: String, content: String, isContentSingleLine: Bool = false, action: @escaping () -> Void) -> some View {
```